addons/web/static/src/core/colorpicker/colorpicker.js

```diff
@@ -94,6 +94,9 @@ export class Colorpicker extends Component {
             this.start();
         });
         onWillUpdateProps((newProps) => {
+            if (!this.el) {
+                return;
+            }
```

Comment:
@dmo-odoo , this is the commit that solved the bug https://github.com/odoo/odoo/commit/91293fe4a8125f65cd7e5f487aacbc62c35c0f74, I don't think it's possible to revert it, but from what I understand from this comment https://github.com/odoo/odoo/blob/aaff9c748e91f78a3c79b0e671a770e8be30532d/addons/web/static/src/core/colorpicker/colorpicker.js#L66-L83 is that the behavior of having `this.el` as `undefined` is possible, I don't know if I'm missing something here.

Reply:
Ok I see, thanks for checking! Then maybe just add this comment:
```js
// There is legacy code that can trigger the instantiation of the
// colorpicker when one of it's parent component is not in the dom.
// If that parent element is not in the dom, owl will not return
// `this.linkComponentWrapperRef.el` because of a check (see
// `inOwnerDocument`).
```
We don't need the TODO since it's already been fixed in the latest version.